src/prime_filed.cpp

```diff
@@ -57,13 +57,10 @@ int main(int argc, char** argv) {
   std::tie(drain_seconds, shutdown_seconds) = parse_quiesce_config(argc > 3 ? argv[3] : "");
   quiesce(drain_seconds, shutdown_seconds);
 
-  // default to no health check, if one is provided its just the path and the canned response is OK
-  http_server_t::health_check_matcher_t health_check_matcher{};
-  std::string health_check_response;
+  shortcircuiter_t<http_request_t> shortcircuiter;
   if (argc > 4) {
-    health_check_matcher = [&argv](const http_request_t& r) -> bool { return r.path == argv[4]; };
-    // TODO: make this configurable
-    health_check_response = http_response_t{200, "OK"}.to_string();
+    uint8_t mask = http::get_method_mask(argv[5]);
```

Comment:
this is not quite right, here you dont check that a 6th argument (ie `argv[5]`) exists. indexing `argv` in this context is then undefined behavior. i would suggest changing this to:

```suggestion
    uint8_t mask = argc > 5 ? http::get_method_mask(argv[5]) : ALL_VERBS_MASK;
```